scripts/screens/ListScreen.py

```diff
@@ -538,6 +538,19 @@ def update_search_cats(self, search_text):
         """Run this function when the search text changes, or when the screen is switched to."""
         self.current_listed_cats = []
         Cat.sort_cats(self.full_cat_list)
+
+        # adding in the guide if necessary, this ensures the guide isn't affected by sorting as we always want them to
+        # be the first cat on the list
+        if self.current_group == 'df' and game.clan.instructor.df:
```

Comment:
You can probably turn this into one if statement; like if (group == df and instructor.df) or (group == sc and not instructor.df) 
Instead of repeating the same code there twice